cli/tests/error_028_data_error_format_js.ts

```diff
@@ -0,0 +1,5 @@
+(async () => {
+  await import(
+    "data:application/javascript;base64,Y29uc29sZS5lcnJvcihFcnJvcigiYmFkIikpOw=="
```

Comment:
I think we should be a lot more generous as to how many characters are allowed before truncation happens. How about >120?

Reply:
Added a threshold of 150 to match `SOURCE_ABBREV_THRESHOLD`